xbmc/media/MediaType.h

```diff
@@ -37,6 +37,7 @@ using MediaType = std::string;
 #define MediaTypeTvShow           "tvshow"
 #define MediaTypeSeason           "season"
 #define MediaTypeEpisode          "episode"
+#define MediaTypeShoutcast        "shoutcast"
```

Comment:
not sure what fits best... "internetradio" would certainly be better already. Or "audiostream" perhaps? or just use MediaTypeMusic, which already exists? (it is currently unused I think)